### PR TITLE
Update hover state on positions page to better indicate how to interact with the heatmap

### PR DESCRIPTION
**Pull request by Mysrt**
also a small change to the readme

edit: and add swap files to the gitignore.  I can break this up if it's a problem.

tested this locally, didn't add any automated tests because the change is so trivial but i can look into it if it's a sticking point.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +67,9 @@ Quickstart
 ----
 * Install dependencies.  If on Debian/Ubuntu: `sudo bash init.sh`  Otherwise, you're responsible for figuring out how to install dependencies yourself.
 * Create .env file with required config values in KEY=VALUE format (see config.js for a full listing of options) `cp .env_example .env`
+  * Note: If you have 'Steam Guard' activated on your account you will
+    either have to deactivate it or create a new account for use with
+    the api, (recommended).
```
Comment (howardchung) [inline README.md]: no comma needed, and it's not technically an API.  I'd just say "for use with the retriever"

**Comment by howardchung** [comment]
We don't have any automated tests for UI so don't worry about that.  It would be nice to have them but our UI is pretty fluid so it may be annoying to keep them up to date.

File: README.md [hunk 2]
```diff
@@ -67,6 +67,9 @@ Quickstart
 ----
 * Install dependencies.  If on Debian/Ubuntu: `sudo bash init.sh`  Otherwise, you're responsible for figuring out how to install dependencies yourself.
 * Create .env file with required config values in KEY=VALUE format (see config.js for a full listing of options) `cp .env_example .env`
+  * Note: If you have 'Steam Guard' activated on your account you will
+    either have to deactivate it or create a new account for use with
+    the retriever.
```
Comment (howardchung) [inline README.md]: you can put back the recommended part, it is definitely recommended not to use your "real" account.

**Comment by howardchung** [comment]
@albertcui please review

File: README.md [hunk 3]
```diff
@@ -67,6 +67,9 @@ Quickstart
 ----
 * Install dependencies.  If on Debian/Ubuntu: `sudo bash init.sh`  Otherwise, you're responsible for figuring out how to install dependencies yourself.
 * Create .env file with required config values in KEY=VALUE format (see config.js for a full listing of options) `cp .env_example .env`
+  * Note: If you have 'Steam Guard' activated on your account you will
+    either have to deactivate it or create a new account for use with
+    the retriever, (recommended).
```
Comment (howardchung) [inline README.md]: Sorry, I meant drop the comma but keep the (recommended).  It's not a big deal, I can probably fix it later myself

**Comment by howardchung** [comment]
signoff

**Comment by albertcui** [comment]
Awesome! Thanks for the contribution. I was thinking about this myself, didn't realize it was a simple CSS fix! (Shows how much I know about front end haha...)

**Comment by albertcui** [comment]
Regression broke this change.

**Pull request merged**